### ISSUE TITLE
[ISSUE] Wrong indentation in webModeler.websockets.resources definition

### ISSUE THREAD

**Issue by lukas-beumer** [camunda/camunda-platform-helm#1009]
**Describe the issue:**

Defining resources for the webModeler.websockets component is not possible because it is not indented correctly in the template. As with the others, it should be indented by 12 digits, not just 10.

**Actual behavior:**

The indentation is defined with 10 spaces.

**Expected behavior:**

The indentation is defined with 12 spaces so that the resource definitions are assumed correctly.

**How to reproduce:**

https://github.com/camunda/camunda-platform-helm/blob/89523dfba3d5c19303b8ee174c6df17b1c0bb915/charts/camunda-platform/templates/web-modeler/deployment-websockets.yaml#L66
```
  webModeler:
    websockets:
      resources:
          requests:
            cpu: 500m
            memory: 265Mi
          limits:
            cpu: 1000m
            memory: 512Mi
```
will be 

```
resources:
limits:
  cpu: 1000m
  memory: 512Mi
requests:
  cpu: 500m
  memory: 265Mi
```
in the generated template.

**Environment:**

GKE, self managed

- Platform: GCP
- Helm CLI version: 3.12.0
- Chart version: 8.3.1

**Comment by jessesimpson36** [comment]
This is fixed by #1014

**Comment by jessesimpson36** [comment]
Thanks for reporting the issue.  We just spotted it today.
